rusk-wallet/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rusk-wallet"
-version = "0.2.4"
+version = "0.2.5"
```

Comment:
Needs to be updated in `src/main.rs` as well. I'll open an issue to prevent that, it's about time.